webpages/settings/index.html

```diff
@@ -306,6 +306,41 @@ <h1 class="setting-heading">{{ msg("moreSettings") }}</h1>
               </div>
             </div>
           </div>
+          <div class="setting-body">
+            <div class="addon-topbar">
+              <span class="addon-name"
+                ><img src="../../images/icons/wrench.svg" style="vertical-align: middle; margin-right: 5px" /> {{
```

Comment:
![image](https://user-images.githubusercontent.com/72760579/102620188-a5f9f680-410b-11eb-895f-db1f9e658086.png)
orange icon... you might want to add some style...